Avoid installing META files for labltk and camlp4: Those are provided by their
respective packages now.

Index: findlib-1.5.5/configure
===================================================================
--- configure.orig
+++ configure
@@ -559,7 +559,7 @@
 
 # Generate the META files now.
 
-l="$ldbm dynlink graphics $lnum str threads unix stdlib bigarray ocamldoc $llabltk $lcamlp4 $lobuild $lcomplibs $lbytes $lspacetime"
+l="$ldbm dynlink graphics $lnum str threads unix stdlib bigarray ocamldoc $llabltk $lobuild $lcomplibs $lbytes $lspacetime"
 
 for dir in site-lib-src/*; do
     # We do not really know if $dir is a directory.
